xcode/Subconscious/Shared/UIKit/BlockEditor/BlockEditorModel.swift

```diff
@@ -137,12 +137,9 @@ extension BlockEditor {
         /// have in cache.
         case refreshTranscludesFor(id: UUID)
         /// Save a snapshot
-        case save(_ snapshot: MemoEntry?)
+        case requestSave(_ snapshot: MemoEntry?)
+        case forwardRequestSave(_ snapshot: MemoEntry)
```

Comment:
What is forwardRequestSave?

Reply:
Since `requestSave` can be emitted as a flow-on from other actions it might not actually be emitted on the `actions` stream of `Store<BlockEditorModel>`. `forwardRequestSave` is an explicitly emitted action, as an effect, to ensure this request is always forwarded up to the app-level.